src/Propel/Common/Config/XmlToArrayConverter.php

```diff
@@ -87,7 +87,7 @@ protected static function simpleXmlToArray(SimpleXMLElement $xml): array
 
             // if it's not an array, then it was empty, thus a value/string
             if ($child === []) {
-                $child = static::getConvertedXmlValue($v);
+                $child = self::getConvertedXmlValue($v);
```

Comment:
why do we need this change?

Reply:
Private prop

Reply:
One of the static analyzers complains about using static on a private or protected method.  @dereuromark and I already discussed this.